tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -453,3 +453,47 @@ def test_torch(self, tol, features, pad_with, normalize):
         res2 = circuit2(features, pad_with)
 
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
+
+
+@pytest.mark.jax
+def test_jacobian_with_and_without_jit_has_same_output_with_high_shots():
+    import jax
+
+    dev = qml.device("default.qubit", wires=4, shots=10000, seed=7890234)
+
+    @qml.qnode(dev)
+    def circuit(coeffs):
+        qml.AmplitudeEmbedding(coeffs, normalize=True, wires=[0, 1])
+        return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+    params = jax.numpy.array([0.4, 0.5, 0.1, 0.3])
+    jac_fn = jax.jacobian(circuit)
+    jac_jit_fn = jax.jit(jac_fn)
+
+    jac = jac_fn(params)
+
+    jac_jit = jac_jit_fn(params)
+
+    assert qml.math.allclose(jac, jac_jit, atol=0.02)
+
+
+@pytest.mark.jax
+def test_jacobian_with_and_without_jit_has_same_output_with_analytic_mode():
```

Comment:
Could combine this test with the one above, but that's a matter of taste. If not, the docstrings of the two tests (also in the next file) should be somewhat different in order to not rely on the test function name alone :)

Reply:
(using `pytest.mark.parametrize`)

Reply:
It's not so important, but one could use a joint parametrization
```python
@pytest.mark.parametrize("shots, atol", [(None, 1e-8), (10000, 0.02)])
```
the seed can be set for the `shots=None` device as well afaik, and the `diff_method` actually is `"parameter-shift"` in both cases, just that it has to be set manually for `shots=None` and is obtained by the internal logic for `shots!=None` :) 

But again, this is not terribly important, was just a suggestion.

Reply:
I see, thank you for the explanation :)